examples/platforms/simulation/platform-simulation.h

```diff
@@ -158,6 +161,25 @@ void platformRadioDeinit(void);
  */
 void platformRadioReceive(otInstance *aInstance, uint8_t *aBuf, uint16_t aBufLength);
 
+/**
+ * This function signals Tx Done to the radio (instead of echo frames).
+ *
+ * @param[in]  aInstance  A pointer to the OpenThread instance.
+ * @param[in]  pktSeq     Packet sequence number to be confirmed as sent.
+ *
+ */
+void platformRadioTxDone(otInstance *aInstance, uint8_t pktSeq);
+
+/**
+ * This function inputs the parameter used to simulate a CCA procedure.
+ *
+ * @param[in]  aInstance  A pointer to the OpenThread instance.
+ * @param[in]  channel    Channel number in use.
+ * @param[in]  value      CCA result (0 for no activity, 255 busy).
```

Comment:
Are values other than 0/255 defined?

Reply:
At the moment, these values are set to 0 or 255, since there is no model for signal propagation in the channel. So, for a future implementation of this property, I already left it using values between 0/255.